### ISSUE TITLE
Example Project

### ISSUE THREAD

**Issue by dfseifert** [Vrekt/Stellar#1]
### Missing Example Project

Hey There!

I've stumbled over your engine and i really like it so thanks for that!
**I just think that your repo is missing an example project.** 

Kind regards

**Comment by Vrekt** [comment]
I agree, I'm not actively working on Stellar but I'll see what I can do about this.
